lib/job-simulation.rb

```diff
@@ -1,31 +1,53 @@
-require './Stack.rb'
-require './Queue.rb'
+require_relative './Stack.rb'
+require_relative './Queue.rb'
 
 class JobSimulation
   attr_reader :workers, :waiting, :roll
 
   def initialize (jobs_available, job_seekers)
-
+    count = 1
+    @workers = Stack.new
+    jobs_available.times do
+      @workers.push("Worker ##{count}")
+      count += 1
+    end
+    @waiting = Queue.new
+    job_seekers.times do
+      waiting.enqueue("Worker ##{count}")
+      count += 1
+    end
   end
 
   def cycle
-
+    # roll, fire, hire
+    roll = rand(5) + 1
```

Comment:
Also I believe `rand(5)` will give you a number 0-4.
Adding `+ 1` at the end will shift your range to 1-5.
I don't think it's possible for your solution to roll a 6.